src/main/scala/com/timushev/sbt/updates/UpdatesPlugin.scala

```diff
@@ -13,10 +13,11 @@ object UpdatesPlugin extends AutoPlugin {
   override val projectSettings = Seq(
     dependencyUpdatesReportFile := target.value / "dependency-updates.txt",
     dependencyUpdatesExclusions := DependencyFilter.fnToModuleFilter(_ => false),
+    dependencyUpdatesFilter := DependencyFilter.fnToModuleFilter(_ => true),
```

Comment:
@rtimush hmmm, I can't seem to get the implicit going. 

<img width="576" alt="screen shot 2017-04-02 at 8 19 17 am" src="https://cloud.githubusercontent.com/assets/1640671/24587060/339463e8-177d-11e7-9e22-d09d2e4adfa9.png">

This doesn't compile, either. 

The only implicit in DependencyFilter.scala that I'd see working here is the `implicit def fnToModuleFilter`. There doesn't seem to be anything that converts "down" from a DependencyFilter to a ModuleFilter, that I can find. Thoughts?